source/MaterialXRenderGlsl/TextureBaker.h

```diff
@@ -118,6 +118,12 @@ class MX_RENDERGLSL_API TextureBaker : public GlslRenderer
         return _outputImagePath;
     }
 
+    /// Set the "libraries" search path location.
```

Comment:
As above, the string "libraries" is no longer required in code search paths, so I would recommend double-checking that this is working as expected.